src/rosdistro/util.py

```diff
@@ -0,0 +1,51 @@
+# Software License Agreement (BSD License)
+#
+# Copyright (c) Microsoft Corporation.
```

Comment:
Since you copy-n-pasted the code snippet I shared from `vcstool` claiming copyright is certainly not right.